jkube-maven-plugin-build/src/main/java/org/eclipse/jkube/maven/plugin/build/JKubeMavenPluginResourceGeneratorBuilder.java

```diff
@@ -18,7 +18,7 @@ public class JKubeMavenPluginResourceGeneratorBuilder {
     private static final Logger logger = Logger.getLogger(JKubeMavenPluginResourceGeneratorBuilder.class.getName());
 
     private Path pom;
-    private String[] k8sGoals = new String[] {"package", "k8s:build", "k8s:resource", "k8s:deploy"};
+    private String[] k8sGoals = new String[] {"package", "k8s:resource", "k8s:build", "k8s:push", "k8s:deploy"};
```

Comment:
May I know the reason of changing the order of `k8s:resource` and `k8s:build` ?

Another comment may not relate to this PR, but not sure if `package` should be activated by default, I see the wildfly-jar-plugin generates the bootable jar 3 times in the log. The 3rd one may come from `k8s:deploy`, maybe consider to `k8s:apply` ?

Reply:
> May I know the reason of changing the order of k8s:resource and k8s:build ?

It seemed to me it was resembly the actual plugin workflow, but I've reverted it now.

> Another comment may not relate to this PR, but not sure if package should be activated by default, I see the wildfly-jar-plugin generates the bootable jar 3 times in the log. The 3rd one may come from k8s:deploy, maybe consider to k8s:apply ?

Yes, apply is actually the right choice if `k8s:build` and `k8s:resource` are executed previously, hence I reverted this change as well.
Regarding `package`, instead, IIUC it's better to have it in the defaults because it's not implied that the required build would happen for sure first.